src/tribler-core/tribler_core/modules/tunnel/tests/test_triblertunnel_community.py

```diff
@@ -22,25 +21,28 @@
 
 from tribler_core.modules.tunnel.community.triblertunnel_community import PEER_FLAG_EXIT_HTTP, TriblerTunnelCommunity
 from tribler_core.tests.tools.base_test import MockObject
+from tribler_core.tests.tools.test_as_server import BaseTestCase
 from tribler_core.tests.tools.tools import timeout
 from tribler_core.tests.tools.tracker.http_tracker import HTTPTracker
 from tribler_core.utilities.path_util import mkdtemp
 from tribler_core.utilities.utilities import succeed
 
 
-class TestTriblerTunnelCommunity(TestBase):
+class TestTriblerTunnelCommunity(TestBase, BaseTestCase):  # pylint: disable=too-many-public-methods
```

Comment:
This is not something that should change right now, but more of a note/memo: `BaseTestCase` seems to have high duplication with `TestBase` and should probably be consolidated.